pkg/testutil/componentsuite/componentsuite.go

```diff
@@ -130,88 +127,20 @@ func runApply(t *testing.T, comp *bundle.Component, tc *TestCase) *bundle.Compon
 	applyOpts := &filter.Options{}
 	applyOpts.Annotations = tc.Apply.Filters
```

Comment:
Unused?

Reply:
Removed. nice catch. Although it exposes an inconsistency in the schema -- removed that too.